File: Solutions/2161. Partition Array According to Given Pivot/2161.py
class Solution:
    def pivotArray(self, nums: List[int], pivot: int) -> List[int]:
        less = []
        great = []
        equal = 0
        for i in nums :
            if i < pivot:
                less.append(i)
            elif i == pivot:
                equal += 1
            else:
                great.append(i)
        for i in range(equal):
            less.append(pivot)
        for i in great:
            less.append(i)
        return less

File: Solutions/2161. Partition Array According to Given Pivot/2161.go
func pivotArray(nums []int, pivot int) []int {
    var less []int
    var great []int 
    equal := 0
    for _, num := range nums {
        if num < pivot {
            less = append(less, num)
        } else if num == pivot{
            equal++
        } else {
            great = append(great, num)
        }
    }
    for range equal {
        less = append(less, pivot)
    }
    for _, num := range great {
        less = append(less, num)
    }
    return less
}